week-3/Homework/mandatory/6-lane-names.js

```diff
@@ -4,10 +4,11 @@
   Write a function that will return all street names which contain 'Lane' in their name.
 */
 
-function getLanes() {
+function getLanes(streetNames) {
+    return streetNames.filter(Element => Element.indexOf("Lane") >=0);
```

Comment:
Like exercise 3, how could we improve this code (hint: try not using `.indexOf()`)